### ISSUE TITLE
Popup forms fails Lighthouse accessibility test: ARIA IDs are not unique

### ISSUE THREAD

**Issue by joethseo** [soflyy/breakdance-bugs#375]
See here:

<img width="988" alt="Screenshot of Google Chrome (13-09-2022, 10-50-14)" src="https://user-images.githubusercontent.com/66306824/189870679-a5e4d27a-c20d-4c73-bbbe-af0745881e7e.png">

This was taken from this URL:

https://web.dev/measure/?gclid=CjwKCAjwsfuYBhAZEiwA5a6CDPPKsunL1SImhOMv7nXi_zV1M8Nl-gWo_721IjMaCuC-ZwB2n_EfrhoC2OEQAvD_BwE&url=https%3A%2F%2Fwww.tribeseo.com%2F#

**Comment by louisreingold** [comment]
Thanks for the report. We've addressed this in a Breakdance but forgot to close the issue.
